.github/workflows/refapp-3x-clinical-visit.yml

```diff
@@ -18,13 +21,11 @@ jobs:
           node-version: '12.x'
       - name: Installing dependencies
         run: npm install
```

Comment:
Isn't  the dependencies required to be installed within the bdd-tests module?

Reply:
```
    defaults:
      run:
        working-directory: ./qaframework-bdd-tests
```
Because of this `working-directory` configuration, all the `run` commands run inside the `qaframework-bdd-tests` directory in default. So we don't have to navigate manually.

(If we use `cd` commands instead of setting the default working directory, we will have to add the navigating command inside every `run:` command. Otherwise it will run inside the root directory in every `run:`)

Reply:
> Because of this working-directory configuration, all the run commands run inside the qaframework-bdd-tests directory in default

This is a nice approach.

Have you tested out this test locally and its passing the build plan?

Reply:
Yeah